### ISSUE TITLE
First programming memories

### ISSUE THREAD

**Issue by dawsbot** [lencioni/ama#11]
What language did you start with, what was the first "aha" moment, or what got you addicted?

**Comment by lencioni** [comment]
It was quite a long time ago, but I believe that my first language was [logo](https://en.wikipedia.org/wiki/Logo_%28programming_language%29). I picked it up in elementary school and learned how to make the turtle do a lot of things. I spent part of the school year teaching it to draw the flags of the world, which was a pretty neat exercise for me at the time.

Shortly after that, I discovered on our home computer a thing called [QBasic](https://en.wikipedia.org/wiki/QBasic)--although at the time it may have actually been [QuickBasic](https://en.wikipedia.org/wiki/QuickBASIC), I don't remember clearly anymore. Logo was kinda fun, but pretty limited in its scope, so it wasn't that addictive for me. When I started writing basic, I got pretty hooked pretty quickly. I think it was because it just felt like magic.

I had a friend who lived nearby who was also interested in computers, so we would teach each other what we learned about QBasic, which was a lot of fun for me at the time. I eventually bought a book about QBasic, which helped us learn a lot faster.

When we eventually got the Internet, I was able to get a newer version that could compile my code into a standalone executable, which was pretty exciting. It made it so I could more easily share my programs with other people. There were also a lot of great examples of programs to look at and learn from, which really helped me grow.

I then became very interested in building web pages and also GUI programs. So, I dabbled in HTML, and other languages like Visual Basic and a really obscure language called [C--](https://en.wikipedia.org/wiki/C--). After a while, I was only interested in web-related languages, and spent a lot of time writing PHP.
